src/test/scala/com/amazon/deequ/anomalydetection/seasonal/HoltWintersTest.scala

```diff
@@ -207,6 +207,73 @@ class HoltWintersTest extends AnyWordSpec with Matchers {
       anomalies should have size 3
     }
   }
+
+  "work on hourly data with daily seasonality" in {
+    // https://www.kaggle.com/datasets/fedesoriano/traffic-prediction-dataset
+    val hourlyTrafficData = Vector[Double](
+      15, 13, 10, 7, 9, 6, 9, 8, 11, 12, 15, 17, 16, 15, 16, 12, 12, 16, 17, 20, 17, 19, 20, 15,
+      14, 12, 14, 12, 12, 11, 13, 14, 12, 22, 32, 31, 35, 26, 34, 30, 27, 27, 24, 26, 29, 32, 30, 27,
+      21, 18, 19, 13, 11, 11, 11, 14, 15, 29, 33, 32, 32, 29, 27, 26, 28, 26, 25, 29, 26, 24, 25, 20,
+      18, 18, 13, 13, 10, 12, 13, 11, 13, 22, 26, 27, 31, 24, 23, 26, 26, 24, 23, 25, 26, 24, 26, 24,
+      19, 20, 18, 13, 13, 9, 12, 12, 15, 16, 23, 24, 25, 24, 26, 22, 20, 20, 22, 26, 22, 21, 21, 21,
+      16, 18, 19, 14, 12, 13, 14, 14, 13, 20, 22, 26, 26, 21, 23, 23, 19, 19, 20, 24, 18, 19, 16, 17,
+      16, 16, 10, 9, 8, 7, 9, 8, 12, 13, 17, 14, 14, 14, 14, 11, 15, 13, 12, 17, 18, 17, 16, 15, 13,
```

Comment:
Can you remove the last `,` ? The build is failing as a result.

Reply:
Fixed